conekta_plugin.php

```diff
@@ -162,4 +166,61 @@ public static function update_conekta_order_meta( $order, $order_value, string $
         $order->save();
     }
 
+    public static function get_user_locale() {
+        $current_user_id = get_current_user_id();
+
+        if ($current_user_id) {
+            $user_locale = get_user_meta($current_user_id, 'locale', true);
+    
+            if (!empty($user_locale)) {
+                return in_array(substr($user_locale, 0, 2), ['es', 'en']) ? substr($user_locale, 0, 2) : 'es';
+            }
+        }
+    
+        $site_locale = substr(get_locale(), 0, 2);
+    
+        return in_array($site_locale, ['es', 'en']) ? $site_locale : 'es';
+    }
+    public static function get_user_ip(): string {
+        $ip_keys = [
+            'HTTP_X_FORWARDED_FOR',   // Load Balancer / Proxies
+            'HTTP_CF_CONNECTING_IP',  // Cloudflare
+            'HTTP_X_REAL_IP',         // Nginx
+            'HTTP_CLIENT_IP',         // Proxy
+            'REMOTE_ADDR'             // Último recurso
```

Comment:
```suggestion
            'REMOTE_ADDR'             // Último recurso
```
Cambiar comentario a inglés